src/main/java/com/comphenix/protocol/utility/MinecraftVersion.java

```diff
@@ -40,7 +40,7 @@ public final class MinecraftVersion implements Comparable<MinecraftVersion>, Ser
     /**
      * Version 1.20 - the trails and tails update
      */
-    public static final MinecraftVersion TRAILS_AND_TAILS = new MinecraftVersion("1.20");
+    public static final MinecraftVersion TRAILS_AND_TAILS = new MinecraftVersion("1.20.1");
```

Comment:
if 1.20.1 requires special handling, it should be a new field. in any case, this should stay 1.20